db/migrate/20221129161022_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :comment_counter, default: 0
+      t.integer :like_counter, default: 0
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] This table should refer to `:author`, this will create author_id and a corresponding index and then the foreign_key to the users' table. Please kindly fix that 🙃 😉